### PR TITLE
feat(Integration): Truera Integration

### PR DESCRIPTION
**Pull request by vivekbhalgat**
# Summary

Create a PR for Truera integration with New Relic.

There was some issue fixing comments in previous pr and so I have created new PR: 
https://github.com/newrelic/newrelic-quickstarts/pull/537

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work? NA
- [ ] Did you check your dashboard image quality? -  Do they look good?  NA
- [ ] Did you check that your alerts actually work?  NA
- [ ] Did you include an InstallPlan and Documentation reference?  Yes(The links need to be updated once I get final version from @iazam)
- [ ] Did you check your descriptive content for voice, tone, spelling and grammar errors?  Yes
- [ ] Did you attach images of your dashboards to the PR so we can see them working?  NA, No dashbards are created, there would be a different tool to setup dashboards

### Screenshots

Attach images of any visual changes, such as a dashboard here.

### CODE REVIEW

File: quickstarts/mlops/truera/dashboards/truera.json [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+            "width": 10
+          },
+          "title": "Model Instability(NUMERICAL_WASSERSTEIN)",
+          "rawConfiguration": {
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 3206889,
```
Comment (orico) [inline quickstarts/mlops/truera/dashboards/truera.json:89]: @vivekbhalgat all of these should be change to "accountId": 0,

Comment (vivekbhalgat) [reply quickstarts/mlops/truera/dashboards/truera.json]: I have updated the json.

File: install/mlops/truera/install.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+id: truera-mlops
+name: Truera Inc
+
+title: Truera diagnositcs and monitoring solution for Trustworthy AI.
+
+description: |
+  TruEra provides best in class AI Quality diagnostics and monitoring solution. With this integration, TruEra and NewRelic users can go beyond standard accuracy, input and output drift monitoring by identifying consequential data drift, and other AI Quality diagnostics such as fairness, conceptual soundness and segment behavior. When combined with NewRelic’s comprehensive observability capabilities, MLOps teams can correlate signals throughout the ML lifecycle.
```
Comment (jpvajda) [inline install/mlops/truera/install.yml:7]: This is a pretty long run on sentence, would you mind trying to break this up into shorter sentences?

File: quickstarts/mlops/truera/config.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+# Authors of the quickstart (required)
+authors:
+  - Truera
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - mlops
+  - ml
+  - ai
+  - machine-learning
```
Comment (jpvajda) [inline quickstarts/mlops/truera/config.yml:27]:
```suggestion

```

File: quickstarts/mlops/truera/config.yml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  - machine learning
+  - ml monitoring
+  - machine learning monitoring
+  - xai
+  - explainable ai
+  - explanability
```
Comment (jpvajda) [inline quickstarts/mlops/truera/config.yml:33]:
```suggestion

```

File: quickstarts/mlops/truera/config.yml [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+  - fairness
```
Comment (jpvajda) [inline quickstarts/mlops/truera/config.yml:34]:
```suggestion

```

File: quickstarts/mlops/truera/config.yml [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+# Displayed in the UI
+title: Truera
+
+# Long-form description of the quickstart (required)
+description: |
+  TruEra provides best in class AI Quality diagnostics and monitoring solution. With this integration, TruEra and NewRelic users can go beyond standard accuracy, input and output drift monitoring by identifying consequential data drift, and other AI Quality diagnostics such as fairness, conceptual soundness and segment behavior. When combined with NewRelic’s comprehensive observability capabilities, MLOps teams can correlate signals throughout the ML lifecycle.
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
```
Comment (jpvajda) [inline quickstarts/mlops/truera/config.yml:12]: I'd recommend a more simplified summary as this is used in the preview tile on the main I/O catalog page.

File: quickstarts/mlops/truera/config.yml [hunk 7]
```diff
@@ -0,0 +1,50 @@
+# Name of the quickstart (required)
+name: truera
+
+# Displayed in the UI
+title: Truera
+
+# Long-form description of the quickstart (required)
+description: |
+  TruEra provides best in class AI Quality diagnostics and monitoring solution. With this integration, TruEra and NewRelic users can go beyond standard accuracy, input and output drift monitoring by identifying consequential data drift, and other AI Quality diagnostics such as fairness, conceptual soundness and segment behavior. When combined with NewRelic’s comprehensive observability capabilities, MLOps teams can correlate signals throughout the ML lifecycle.
```
Comment (jpvajda) [inline quickstarts/mlops/truera/config.yml:9]: This is a pretty long run on sentence, would you mind trying to break this up into shorter sentences?

**Review by jpvajda — CHANGES_REQUESTED** [review]
I made some suggestions. Thanks for the contribution!

**Comment by jpvajda** [comment]
@vivekbhalgat We just need you to sign the [CLA](https://github.com/newrelic/newrelic-quickstarts/pull/566#issuecomment-969569203) to accept this other contribution, once you do it you will no longer be requiredt so sign for future PRs. Thanks!

File: install/mlops/truera/install.yml [hunk 8]
```diff
@@ -0,0 +1,16 @@
+id: truera-mlops
+name: Truera Inc
+
+title: Truera diagnositcs and monitoring solution for Trustworthy AI.
```
Comment (ahaberkoExt) [inline install/mlops/truera/install.yml:4]: Correct the typo in `diagnositics` > `diagnostics`.
You can also maintain more consistency by keeping the lowercase on `trustworthy`.

File: install/mlops/truera/install.yml (hunk 2 above)
Comment (ahaberkoExt) [reply install/mlops/truera/install.yml:7]: You can try: 
`TruEra provides best in class AI quality diagnostics and monitoring solution.`

File: quickstarts/mlops/truera/config.yml (hunk 7 above)
Comment (ahaberkoExt) [reply quickstarts/mlops/truera/config.yml:9]: You can try:
`Use the TruEra integration with New Relic and go beyond standard accuracy, input and output drift monitoring and identify consequential data drift and other AI quality diagnostics.`

File: quickstarts/mlops/truera/config.yml (hunk 6 above)
Comment (ahaberkoExt) [reply quickstarts/mlops/truera/config.yml:12]: You can try:
`Go beyond standard accuracy, input and output drift monitoring and identify consequential data drift and other AI quality diagnostics as well as correlate signals throughout the machine learning lifecycle.`

File: quickstarts/mlops/truera/config.yml (hunk 4 above)
Comment (ahaberkoExt) [reply quickstarts/mlops/truera/config.yml:33]: Did you mean `explainability`?

**Review by ahaberkoExt — COMMENTED** [review]
Hi all, just reviewing the language here - found a few typos, and proposed shortened descriptions. Feel free to use these.

File: quickstarts/mlops/truera/config.yml (hunk 4 above)
Comment (jpvajda) [reply quickstarts/mlops/truera/config.yml:33]: yes! I'll fix this suggestion

**Comment by jpvajda** [comment]
To move this along, I'll pull down this PR and make the suggested changes.

**Comment by jpvajda** [comment]
closing and will resolve in #606

**Pull request closed without merging**